draft-ietf-rats-corim.md

```diff
@@ -1595,17 +1603,18 @@ Profile (label 6):
 ~~~ cddl
 ECT = {
   ? e: environment-map
-  ? c: claims-map / [ + local-claim ]
+  ? c: [ + local-claim ]
```

Comment:
Line 1583 which describes the claims needs to be:
1. Reworded with a more appropriate name, like a set of claims 
2. Needs to explain better, regarding the claims pertaining to each of the measured elements belonging to a target environment

Reply:
Fixed in commit: https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/268/commits/59fd80d9894bf52b9a6718d9339d6548c238ee9f